src/store/ducks/opensrp/jurisdictions/index.ts

```diff
@@ -0,0 +1,149 @@
+import { get, keyBy, values } from 'lodash';
```

Comment:
@p-netm what is the reasoning for not simply using the existing `src/store/ducks/jurisdictions.ts`?

Reply:
Mosh thought this should go here, i think he mentioned something to do with portability of this ducks module to other FE apps that call to opensrp,.. i am not sure though if this was the reason, @moshthepitt, please help me answer this :point_up:

Reply:
> portability of this ducks module to other FE apps that call to opensrp

Ok yeah that makes sense 👍